spec.md

```diff
@@ -478,6 +484,7 @@ Location: <location>
 The `<location>` is a pullable manifest URL.
 The Docker-Content-Digest header returns the canonical digest of the uploaded blob, and MUST be equal to the client provided digest.
 Clients MAY ignore the value but if it is used, the client SHOULD verify the value against the uploaded blob data.
+If the client provided digest is invalid or uses an unsupported algorithm, the registry SHOULD respond with a response code `400 Bad Request`.
```

Comment:
It should probably be mentioned here that the error code of a potential body must be `code-4`. There's nothing more annoying and hard to debug than a 400 without any indication of what went wrong.

Reply:
I agree that it would be useful to document which error message should be returned, but we haven't done that anywhere else in the spec, so I think that is best done in a separate PR.